examples/train_model.py

```diff
@@ -101,6 +116,10 @@ def main():
                         type=float, default=1)
     parser.add_argument('-vtim', '--validate-every-n-secs',
                         type=float, default=False)
+    parser.add_argument('-vimp', '--validation-impatience',
```

Comment:
I've seen other people call this `patience`, could we call it that? this is just a long param name :)